src/create-glamorous.js

```diff
@@ -81,6 +87,30 @@ function createGlamorous(splitProps) {
         })(GlamorousComponent.styles)
       }
 
+      function shouldUpdate(props, context) {
```

Comment:
Please take some time to consider how this could be improved. I'm going to merge this into `next` now so we have time to improve it before that's merged into `master` :)